github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1086,7 +1049,7 @@ presubmits:
             - "/usr/local/bin/runner.sh"
             - "/bin/sh"
             - "-c"
-            - "export TARGET=k8s-1.17 && export KUBEVIRT_STORAGE=rook-ceph-default && automation/test.sh"
+            - "export TARGET=k8s-1.20 && export KUBEVIRT_STORAGE=rook-ceph-default && automation/test.sh"
```

Comment:
Could you pls move these env vars to the `env` property of the container? Like here https://github.com/kubevirt/project-infra/blob/8d11e9b6d3d78b16e4da2ffd1b045e3f344c5509/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L1004-L1010

Other than this looks good :+1: